corehq/apps/case_search/views.py

```diff
@@ -64,7 +65,8 @@ def post(self, request, *args, **kwargs):
             value = re.sub(param.get('regex', ''), '', param.get('value'))
             if '/' in param.get('key'):
                 query = '{} = "{}"'.format(param.get('key'), value)
-                search = search.xpath_query(self.domain, query, fuzzy=param.get('fuzzy'))
+                fuzzy_props = set(param.get('key').split('/')[-1:])
+                search = search.xpath_query(self.domain, query, fuzzy_props)
```

Comment:
I might be missing some context on how this is used, can you explain this change?  It looks like before the request defined what was fuzzy and now I guess it's always fuzzy?

Reply:
good point

Reply:
removed fuzzy matching here since we can't really do fuzzy matching on ancestor queries in practice because we don't know the case type of the ancestor: e9a59f7cd4e